app/dashboard/models.py

```diff
@@ -52,6 +53,10 @@
 logger = logging.getLogger(__name__)
 
 
+def get_sentinel_user():
```

Comment:
I think we can remove this function if we are using SET_NULL. @mbeacom

Reply:
@SaptakS Addressed in:  https://github.com/gitcoinco/web/commit/8250d24ae0b4f347425f8e8ba148ade1ba94feaf